### PR TITLE
Update team to spring 2021

### LINKED ISSUE #50: Update information for spring 2021 officer changes
The [team](https://www.acmutd.co/team.html) page needs to be updated to reflect changes in officers who will leave ACM or join ACM after the spring 2021 semester.

Two new directors have been onboarded for the spring 2021:
- Director of Development
- Director of Research

Additionally, a few teams have taken on new officers:
- HackUTD
- Development

For a list of current officers, see [this spreadsheet](https://docs.google.com/spreadsheets/d/1nLhtCfl5kZXiOUN0tNO9wtnxRo-39KjJA6I4qDW_Kkk/edit#gid=630343822). Anyone whose name is not on the "Current ACM Team" sheet needs to be removed from the team page. Titles for officers also need to be updated, as well.

### PR DESCRIPTION
**Pull request by stevenmqnguyen**
Closes #50

### CODE REVIEW

**Comment by harshasrikara** [comment]
Recommendation: Wait till after celebration night to merge this.

File: _officers/hackutd-17-neha.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 name: "Neha Rode"
-role: "Director of HackUTD"
-category: board
-filter: "board,hackutd"
+role: "HackUTD Marketing Lead"
```
Comment (harshasrikara) [inline _officers/hackutd-17-neha.md:3]: @Neha-Rode you might want to update the officer DB lol, looks like it still says you're the HackUTD Marketing Lead 🙃

**Comment by stevenmqnguyen** [comment]
@harshasrikara How can I edit the spreadsheet? My name isn't on there.

**Comment by harshasrikara** [comment]
> @harshasrikara How can I edit the spreadsheet? My name isn't on there.

There's a tab at the bottom called All. Click on that one and you'll be able to update it. There's also more info on it in ACM General > How To Guides > Officer Database

Also at some point I think a lotta values in the spreadsheet fell out of date. Should be revisited later on.

File: _officers/hackutd-17-neha.md (hunk 1 above)
Comment (Neha-Rode) [reply _officers/hackutd-17-neha.md:3]: Updated :))

**Pull request merged**